pkgs/top-level/all-packages.nix

```diff
@@ -33064,4 +33064,6 @@ with pkgs;
   zthrottle = callPackage ../tools/misc/zthrottle { };
 
   zktree = callPackage ../applications/misc/zktree {};
+
+  restinio = callPackage ../development/libraries/restinio {};
```

Comment:
Please sort this into a fitting category and sort there alphabetically.

Reply:
Done.